Getting Started Page

lmilechin opened the pull request:
Initial Getting Started page based on the SuperCloud one: https://supercloud.mit.edu/getting-started.

lmilechin commented:
New Page: https://mit-orcd.github.io/orcd-docs-previews/PR/PR25/getting-started/

lmilechin commented on docs/getting-started.md:
```diff
@@ -0,0 +1,305 @@
+If you don't already have an account, follow the instructions for the system you are interested in:
+
+- Engaging and Satori: Login into the respective OnDemand Portal (below) using your MIT kerberos credentials. The system will then be prompted to create your account automatically. Wait couple minutes for the system to create all the pieces for your account before submitting your first job.
+    - Engaging: [https://engaging-ood.mit.edu]
```
Fix links

lmilechin commented on docs/getting-started.md:
```diff
@@ -0,0 +1,305 @@
+moderate personal laptop or desktop you can request an interactive
+session to run your code in by executing the command:
+
+`LLsub -i`
```
Update with codeblocks/tabs for different system

shaohao99 commented:
The link for [Logging into OpenMind].
https://github.mit.edu/MGHPCC/OpenMind/wiki/How-to-log-in-Openmind%3F

shaohao99 commented:
It would be good to make a pull-down section or a link to a new page for the Linux Command Line section.

lmilechin commented on docs/getting-started.md:
```diff
@@ -0,0 +1,305 @@
+-->
+
+
+Testing your Code
```
Change to "Running your First Job"

lmilechin commented:
I added tabs for the sections where there are different information or links for the different systems. The tabs are synchronized so once you select one they will all be set to the same tab. Pretty cool!

Also updated the OpenMind Logging In link and put the Linux Commands in a collapsible box.

shaohao99 commented:
It looks very nice. I like the selectable and expendable tabs. I think it is good to publish.

lmilechin commented:
Acceptable Use and Code of Conduct page also added: https://mit-orcd.github.io/orcd-docs-previews/PR/PR25/code-of-conduct/

shaohao99 commented:
Would it be good to make "Data Security and Privacy" and "Acceptable Use and Code of Conduct" sub-pages under a bigger title "ORCD policy" (or another similar name)? This will make the contents on the left column shorter, which is good for future topics. 

Other than that, I think it is good to publish.

lmilechin commented:
Added acknowledgements page: https://mit-orcd.github.io/orcd-docs-previews/PR/PR25/acknowledgements/

christophernhill reviewed (approved):
Looks ready to go

shaohao99 commented:
Looks all good to me.
